### PR TITLE
adding final project

### PR DESCRIPTION
**Pull request by branchwelder**
this was really hard

### CODE REVIEW

**Review by segerphilip — COMMENTED** [review]
This looks good, but see these comments.

🚢 🇮🇹

File: ncc1701d.py [hunk 1]
```diff
@@ -0,0 +1,2 @@
+def enterprise():
+    print("engage")
```
Comment (segerphilip) [inline ncc1701d.py:2]: Maybe consider calling this function?

Comment (branchwelder) [reply ncc1701d.py:2]: geez, fine.

Comment (branchwelder) [reply ncc1701d.py:2]: Actually you're completly right. I will do that! Thanks Philip!

Comment (segerphilip) [reply ncc1701d.py:2]: 👍

**Pull request closed without merging**